### PR TITLE
ci, Add .travis.yml ci file

### PR DESCRIPTION
**Pull request by RamLavi**
Signed-off-by: Ram Lavi <ralavi@redhat.com>

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+language: go
+go: "1.13"
+
+services:
+  - docker
+
+install: true
+script:
+  - make docker-test
```
Comment (phoracek) [inline .travis.yml:9]: We already do this on the prow run. Please drop it.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  - provider: script
+    script: docker login -u="$DOCKER_USER" -p="$DOCKER_PASS" quay.io && make docker-build docker-push
+    skip_cleanup: true
+    on:
+      branch: master
+  - provider: script
+    script: docker login -u="$DOCKER_USER" -p="$DOCKER_PASS" quay.io && make docker-build docker-push IMAGE_TAG=$TRAVIS_TAG
+    skip_cleanup: true
+    on:
+      tags: true
```
Comment (phoracek) [inline .travis.yml:21]: Please add a new line

**Review by phoracek — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (RamLavi) [reply .travis.yml:9]: Done

File: .travis.yml (hunk 2 above)
Comment (RamLavi) [reply .travis.yml:21]: Done

**Comment by RamLavi** [comment]
@phoracek fixes [here](https://github.com/kubevirt/ovs-cni/compare/c5c837bd3c2fdb52942da814f96613295ce3ce11..cd9969a5a6707f939cd839e4c10749f3fc927737)

**Review by phoracek — APPROVED** [review]
/lgtm
/approve

Let's see if it works. Secrets should be in place.

**Pull request merged**